manuscript.md

```diff
@@ -2,31 +2,39 @@
+
+Since the original publication of the first peptide identification tool, SEQUEST [24226387], proteomics software has evolved into a sophisticated ecosystem encompassing multiple stages of data processing, advanced predictive models, and robust computational frameworks. 
+Tools like Percolator [17952086] optimize peptide and protein identification through machine learning, while Prosit [31133760] applies deep learning to predict fragment ion intensities, aiding in more accurate spectra matching. Platforms like GalaxyP [29092937], or quantms [38965444] facilitate accessible, reproducible analyses through high-performance computing (HPC) and distributed workflows, supporting researchers in handling large datasets and complex analyses. 
+Together, these advances underscore how proteomics software has transformed into a multidisciplinary field, in a complex ecosystem of algorithms, models, and software tools requiring sophisticated computational and algorithmic expertise.
 
 Computational proteomics, along with algorithm and software development, faces several key challenges common to other omics fields:
 
 - The increasing complexity and size of data acquired by mass spectrometers, the complex sequence of steps, including spectral processing, statistical analysis, and biological interpretation, along with the need to manage algorithmic details and parameter settings, all contribute to making software development in proteomics a complex and demanding endeavor.
 
-- Without structured development practices, proteomics software often struggles to maintain the quality, sustainability, and reproducibility necessary for reliable scientific research. [W: I don't think that OSS doesn't suffer from this.]
+- Ensuring reproducibility is a challenge due to the limited access to detailed algorithmic information, which hinders validation and extension of methods.
 
 - Although most software tools are described in publications, the absence of open-source code, comprehensive documentation, and version control often impedes reproducibility and reuse. 
-This lack of accessibility also limits opportunities for computer scientists [maybe this should be researchers?] and developers to contribute effectively to advancing proteomics software.
+This lack of accessibility also limits opportunities for researchers and developers to contribute effectively to advancing proteomics software. 
+- Custom licenses and restrictions on software distribution can further complicate the situation, making it difficult to share, modify, or redistribute software, and hindering the development of a collaborative and open-source ecosystem.
+Proteomics software is usually on distribution by licences and also platform specific, which makes it difficult to use in a diverse environment and services, putting a restriction to the field and the integration of proteomics with other omics fields. 
```

Comment:
Not sure how to understand "platform specific" in this context.
OS? or more like a software that integrates other tools (e.g., FragPipe)

Reply:
@timosachsenberg can you read it now? Does it read better?